src/microbe_stage/editor/EditorActionHistory.cs

```diff
@@ -135,7 +135,7 @@ public override void AddAction(TAction action)
         // Handle adding directly the action to our history cache, this saves us from having to rebuild the cache
```

Comment:
Does this comment still make sense?

Reply:
It does still in the other (`else`) branch add it to the cache. I wanted to remove the potential bug without having to complicate things a lot (I could grab any actions in the to be queued thing that are after the history reset, but I think history resetting actions are rare so it doesn't require that much optimization).

Reply:
I'll add a comment on the line I changed to clarify that.